.github/tl_packages

```diff
@@ -38,6 +38,7 @@ varwidth
 #
 # special testing for latex-lab
 pdfmanagement-testphase
+l3experimental
```

Comment:
If we need this, shouldn't it be done manually using `texmf`?

Reply:
maybe, but on the other hand I think it is perfectly fine if this is not a "frozen" version but the one that is really on ctan because that is how it is used by others. So In my opinion this is fine.
It is not needed by the code in latex-lab only in Ulrike's code which is called in the test file.